pkgs/applications/emulators/xemu/default.nix

```diff
@@ -0,0 +1,136 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, qemu
+, makeDesktopItem
+, copyDesktopItems
+, pkg-config
+, python3
+, ninja
+, meson
+, which
+, perl
+, wrapGAppsHook
+, glib
+, gtk3
+, libpcap
+, openssl
+, libepoxy
+, libsamplerate
+, SDL2
+, SDL2_image
+, mesa
+, libdrm
+, libGLU
+, gettext
+, vte
+}:
+
+let
+
+
+
+in stdenv.mkDerivation rec {
```

Comment:
Is this right? Seems like one too many lines.

Reply:
yes, fixed it